content/user/starthere/UI-navigation.md

```diff
@@ -51,20 +52,20 @@ If you have system administration capabilities, you will be able to switch from
 
 ## Grid Actions
 
-Many pages within Contrast contain rows of data presented in table format. These grids have a built-in search mechanism to easily get to the data you need. You can also use the quick views to see data by the most frequently used categories. 
+Contrast displays primary assets and findings - applications, servers, libraries, vulnerabilities and attacks - in table format with rows of data. Use the search field to quickly find the data you need. You can also use the quick views to see data by the most frequently used categories. 
 
 <a href="assets/images/Vulnerabilities-grid.png" rel="lightbox" title="View vulnerabilities in the grid"><img class="thumbnail" src="assets/images/Vulnerabilities-grid.png"/></a>
 
-Each table offers column sorting to alter the ordering of the data in exactly the way you want. In addition, tables offer actions that allow you to perform operations on individual rows or in bulk by selecting many rows. 
+In each grid, complete actions to manage your organization - such as merging applications or sending a vulnerability to your bugtracker - for individual or multiple items. For individual items, hover over the grid row, and select the appropriate icon to for each action. For multiple items, use the check marks to select each row, and use the icons in the bottom action bar to complete each action. 
 
 ### Sorting and filtering 
 
-Advanced filtering is available on each main listing of assets and findings - Applications, Servers, Libraries, Vulnerabilities, and Attacks. Simply click the **Advanced** link next to the search field. Here you’ll find panels of various data sets that allow you to limit the result set and hone in on the pieces that interest you.
+Sort and filter data in the grids to adjust your focus. Use the dropdown menu above the grids to select a quick view of findings in certain categories, such as licensed applications or open vulnerabilities. You can also use the filters in the grids to find specific sets of findings, such as servers with the same tags. The dropdown menu above the right corner of the grid lets you sort all the findings in your current view by certain categories, such as application name (alphabetically), in ascending or descending order. 
```

Comment:
You already referred to these as "quick views" earlier in the Grid Actions section, so here I would just say "Use the quick views at the top of the page to hone in on findings in certain categories..."  

"You can also use the column header filters to narrow down to a specific set of findings...."

"The Sort button in the top right corner of the page provides variables upon which you can choose to sort the grid data in ascending or descending order - for example, application name."

Reply:
I wonder if we should mention Select All capability per category? Maybe not, as it's probably pretty easily discoverable.

Reply:
Another option: "You can also use the column header filter panels to locate a specific set of findings..."